kolibri/plugins/coach/assets/src/views/lessons/LessonSummaryPage/sidePanels/LessonResourceSelection.vue

```diff
@@ -8,69 +8,88 @@
     @shouldFocusFirstEl="() => null"
   >
     <template #header>
-      <h1 class="side-panel-title">{{ $tr('manageLessonResourcesTitle') }}</h1>
+      <h1 class="side-panel-title">
+        {{ !showSearch ? $tr('searchFiltersTitle') : $tr('manageLessonResourcesTitle') }}
```

Comment:
This is okay for now, but I think we may end up making some changes here when we update the side panel component a little bit, to make the title here more dynamic. (this is not blocking and just an FYI in case you see changes here later, or I ask you to make them in an upcoming issue 😄)